frontend/src/metabase/admin/tasks/containers/TasksApp/utils.ts

```diff
@@ -42,3 +64,8 @@ function parseStatus(param: QueryParam): UrlState["status"] {
 function isTaskStatus(value: string): value is TaskStatus {
   return ["success", "started", "failed", "unknown"].includes(value);
 }
+
+function parseTask(param: QueryParam): UrlState["task"] {
+  const value = Array.isArray(param) ? param[0] : param;
```

Comment:
I'm seeing this happen enough that I think it would make sense to have a helper for this like we have `isNotNull`.

Maybe `first<T>(x: T | T[]): T | undefined`?

Reply:
I kept it url-state-specific for now

54a749e00d61494d3206538744e6ee258682722d